### PR TITLE
support other CI providers for osx-arm64

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Do we want to swap the default as well?

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
> Do we want to swap the default as well?

Nope. It's really slow due to all the SDKs being in one repo.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
This is ready

File: recipe/download_osx_sdk.sh [hunk 1]
```diff
@@ -19,26 +19,30 @@ if [[ "${MACOSX_SDK_VERSION:-0}" == "0" ]]; then
...
+            export OSX_SDK_DIR_NEW=/Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
         fi
+    fi
+    if [[ ! -d "${OSX_SDK_DIR}/MacOSX${MACOSX_SDK_VERSION}.sdk" ]]; then
+        tmpdir=$(mktemp -d)
+        mkdir -p $tmpdir
+        pushd $tmpdir
+            curl -L -O https://github.com/alexey-lysiuk/macos-sdk/archive/0ecfb46da65f2f1fab77059ebb43de3ac7b0edad.tar.gz
+            tar -xf 0ecfb46da65f2f1fab77059ebb43de3ac7b0edad.tar.gz
+            cp -rf macos-sdk-0ecfb46da65f2f1fab77059ebb43de3ac7b0edad/MacOSX11.0.sdk ${OSX_SDK_DIR}/
```
Comment (xhochy) [inline recipe/download_osx_sdk.sh:36]: Wouldn't `mv` be much faster?

**Comment by xhochy** [comment]
What's the background of this? Which CI do we want to use?

File: recipe/download_osx_sdk.sh (hunk 1 above)
Comment (isuruf) [reply recipe/download_osx_sdk.sh:36]:
```suggestion
            mv macos-sdk-0ecfb46da65f2f1fab77059ebb43de3ac7b0edad/MacOSX11.0.sdk ${OSX_SDK_DIR}/
```

**Comment by isuruf** [comment]
On azure, for osx-64 for gnutls we need macos-10.14 which doesn't have the correct SDK (only 10.15)

**Comment by isuruf** [comment]
@xhochy, @beckermr, @CJ-Wright, this is ready now

**Review by CJ-Wright — APPROVED** [review]
This seems ok, I'm not sure I'm in a good position to review though

**Comment by isuruf** [comment]
Thanks. Let's merge to see if it works. I'll revert if there's any issues

**Comment by jakirkham** [comment]
Thanks Isuru! 😄

**Pull request merged**